RuneWordFinder4/Models/Repository/MongoDataService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using MongoDB.Driver;
+using MongoDB.Bson;
+using System.Collections.Generic;
+using NLog;
+
+namespace RuneWordFinder4.Models.Repository
+{
+    /// <summary>
+    /// https://samueleresca.net/2015/08/querying-mongodb-with-asp-net-mvc/
```

Comment:
Double check this source was used. If not, remove it

Reply:
It was used, comment was cleaned a little. Also removed the TODO below in favor of issue #3.